SPECS/ca-certificates/ca-certificates.spec

```diff
@@ -363,6 +371,21 @@ rm -f %{pkidir}/tls/certs/*.{0,pem}
 %files shared
 %license LICENSE
 
+%files static
+# base files
+%{_datadir}/pki/ca-trust-source/%{p11_format_base_bundle}
+%{_datadir}/pki/ca-trust-legacy/%{legacy_default_base_bundle}
+%{_datadir}/pki/ca-trust-legacy/%{legacy_disable_base_bundle}
+# main pkg files
+%{_datadir}/pki/ca-trust-source/%{p11_format_mozilla_bundle}
+%{_datadir}/pki/ca-trust-legacy/%{legacy_default_mozilla_bundle}
+%{_datadir}/pki/ca-trust-legacy/%{legacy_disable_mozilla_bundle}
+# microsoft files
+%{_datadir}/pki/ca-trust-source/%{p11_format_microsoft_bundle}
+%{_datadir}/pki/ca-trust-legacy/%{legacy_default_microsoft_bundle}
+%{_datadir}/pki/ca-trust-legacy/%{legacy_disable_microsoft_bundle}
```

Comment:
We MUST NOT install all sources for different certificate packs with one subpackage. They are kept separate by design. Usually, a basic Mariner image should only need the ones from the `*-base` subpackage. The expectation is to trust only the minimial set of entities. Expending that trust is up to the user and we cannot implicitly trust anything else for them.